payloads/library/general/Wake-On-Lan/payload.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash 
+LED STAGE1
+NETMODE NAT
+
+#### OPTIONS
+INT='eth0'                #interface of the outgoing interface
```

Comment:
These variable-names are a bit short, could you make them longer (for example INTERFACE instead of INT)?